Formula/apt.rb

```diff
@@ -0,0 +1,113 @@
+class Apt < Formula
+  desc "Advanced Package Tool"
+  homepage "https://wiki.debian.org/apt"
+  url "https://deb.debian.org/debian/pool/main/a/apt/apt_2.3.7.tar.xz"
+  sha256 "c946a126263b3ea51a15216689602367c0da95d6878b76dc891ea335d87bd936"
+  license "GPL-2.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "docbook-xsl" => :build
+  depends_on "doxygen" => :build
+  depends_on "libxslt" => :build
+  depends_on "perl" => :build
+  depends_on "w3m" => :build
```

Comment:
Is this really a build dependency? That's very strange.

Reply:
If I remove `w3m` from the build dependencies, it fails with:
```
cd /tmp/apt-20210811-10856-1uhvmzt/apt-2.3.7/build/doc && ../../CMake/run_if_exists.sh --stdout /tmp/apt-20210811-10856-1uhvmzt/apt-2.3.7/build/doc/design.text /tmp/apt-20210811-1
env: <E2><80><98>w3m<E2><80><99>: No such file or directory
doc/CMakeFiles/libapt-pkg-doc.dir/build.make:264: recipe for target 'doc/design.text.w3m-stamp' failed
```
So I guess we do need it, even if it is an odd dependency.